### PR TITLE
PLG-815 read opertation on product_model : added with_quality_scores parameter

### PR DESCRIPTION
**Pull request by FSAkeneo**
product_model get and list operations now accept parameter 'with_quality_score' as it was the case for products since https://github.com/akeneo/pim-api-docs/pull/427.

### CODE REVIEW

**Comment by olivierpontier** [comment]
It looks your IDE automatically reformated some files. Could you revert it please?

**Comment by olivierpontier** [comment]
It looks your IDE automatically reformated some files. Could you revert it please?

**Review by olivierpontier — APPROVED** [review]
GTM after revert the automatic formatting on products.md and akeneo-web-api.json files

**Comment by FSAkeneo** [comment]
> GTM after revert the automatic formatting on products.md and akeneo-web-api.json files

ok it's done, thanks @olivierpontier

**Review by LaurentPetard — APPROVED** [review]

**Review by antoinetrouve — APPROVED** [review]

File: content/concepts/products.md [hunk 1]
```diff
@@ -814,6 +821,10 @@ Note that the `metadata` field is only available since the 2.3 version and as it
 Note that the `family` field is only available since the 3.2 version.
 :::
 
+::: warning
+Note that the `quality_scores` field is only available on SaaS platforms and when the `with_quality_scores` query parameter is set to `true`.
```
Comment (meganelepalud-akeneo) [inline content/concepts/products.md:825]:
```suggestion
Note that the `quality_scores` field is **only available on SaaS platforms** and when the `with_quality_scores` query parameter is set to `true`.
```

File: content/swagger/parameters.yaml [hunk 2]
```diff
@@ -99,13 +99,19 @@ with_table_select_options:
...
   name: with_quality_scores
   in: query
   description: Return product quality scores in the response. (Only available since the 5.0 version)
   default: false
   type: boolean
   required: false
+with_quality_scores__product_models:
+  name: with_quality_scores
+  in: query
+  description: Return product model quality scores in the response. (Only available on SaaS platforms)
```
Comment (meganelepalud-akeneo) [inline content/swagger/parameters.yaml:112]:
```suggestion
  description: Return product model quality scores in the response. **(Only available on SaaS platforms)**
```

File: content/swagger/resources/product_models/definitions/product_model.yaml [hunk 3]
```diff
@@ -118,6 +118,11 @@ properties:
         description: Status of the product model regarding the user permissions (only available since the v2.3 in the Enterprise Edition)
         type: string
         enum: ['read_only','draft_in_progress','proposal_waiting_for_approval','working_copy']
+  quality_scores:
+    type: object
+    description: Product model quality scores for each channel/locale combination (only available on SaaS platforms and when the "with_quality_scores" query parameter is set to "true")
```
Comment (meganelepalud-akeneo) [inline content/swagger/resources/product_models/definitions/product_model.yaml:123]:
```suggestion
    description: Product model quality scores for each channel/locale combination (**only available on SaaS platforms** and when the "with_quality_scores" query parameter is set to "true")
```

**Review by meganelepalud-akeneo — APPROVED** [review]

File: content/swagger/parameters.yaml (hunk 2 above)
Comment (FSAkeneo) [reply content/swagger/parameters.yaml:112]: It looks the markdown interpreter is not run on the string parts, I use \<strong\> instead.

**Pull request merged**